WidgetExtension/Up Next/EpisodeView.swift

```diff
@@ -62,27 +55,57 @@ struct EpisodeView: View {
     }
 }
 
-struct WidgetPlayToggleStyle: ToggleStyle {
-    func makeBody(configuration: Configuration) -> some View {
-        ZStack {
-            configuration.label
-                .truncationMode(.tail)
+struct WidgetFirstEpisodePlayToggleStyle: ToggleStyle {
+    let colorScheme: PCWidgetColorScheme
 
-            Circle()
-                .foregroundStyle(.white)
-                .frame(width: 24, height: 24)
+    func makeBody(configuration: Configuration) -> some View {
```

Comment:
Should this be marked @ViewBuilder in case the view is not a single View? I think the protocol method is marked ViewBuilder, so maybe that's automatically applied?

Reply:
same as above (tldr; I _**think**_ its fine)